src/Presentation/TabsHtmlBuilder.php

```diff
@@ -47,6 +51,22 @@ private function parseQuery( string $searchQuery ): Query {
 		return $this->queryStringParser->parse( $searchQuery );
 	}
 
+	private function buildSettingsViewModel(): array {
+		$title = Title::newFromText( WikibaseFacetedSearchExtension::CONFIG_PAGE_TITLE, NS_MEDIAWIKI );
+
+		if (
+			!$title instanceof Title
+			|| !$this->user->isAllowed( 'editsitejson' )
+		) {
+			return [];
```

Comment:
The logic here is not quite right

we need to check (wg)`WikibaseFacetedSearchEnableInWikiConfig`. Only show the link if it is `true`

we should not care about if the page already exists or not. If it does not exist, all the more reason for the people to go there and configure things

Lastly, I suspect there is a way to do something like `user->canEdit( $title )`. That is safer, since it checks all the relevant permissions, and avoids needing to know what those are in the `TabsHtmlBuilder`. That `canEdit` method does not exist and I cannot readily find the right one. It might well not be on the `User` class. I suspect we used it _somewhere_ a few months ago.